### PR TITLE
fix(iconButton): missing target prop

### PR DESCRIPTION
**Pull request by KlaudiaBrzywcy**
## Name PR according scheme: type of changes (feat/fix/refactor etc.) + description
Fix icon-Button missing target prop
## Jira issue

JIRA issue: [FSU-102](https://klau.atlassian.net/browse/FSU-102)

## Describe your changes
added missing target prop to IconButton (the same as in HTML like target=_blanc)

## Checklist before requesting a review

- [ ] Checked which branch you would merge.
- [ ] Checked if the code follows the style guidelines of this project (use eslint and prettier).
- [ ] Performed a self-review of code (bugs, clean code rules, etc).
- [ ] Checked if added variables have unique names.
- [ ] Checked if added code/function is not a duplicate. 
- [ ] Tests have been added if possible.
- [ ] New and existing unit tests pass locally with my changes.

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -1,8 +1,10 @@
-NODE_ENV=
```
Comment (im-klau) [inline .env.example:1]: the file is still here 🙈

**Review by im-klau — CHANGES_REQUESTED** [review]

File: .env.example [hunk 2]
```diff
@@ -1,8 +1,10 @@
...
-HOST=
-PORT=
-APP_KEYS=
-API_TOKEN_SALT=
-ADMIN_JWT_SECRET=
-JWT_SECRET=
\ No newline at end of file
+HOST=0.0.0.0
+PORT=1337
+APP_KEYS=IP/TTdeR5D5ZlPgy21CKag==,D1wyCeSyFHgzBqhsqfcoqA==,yJAmEkfS8z/gmVrrmt1ufA==,VH5gF53XJ+IK0/bi9EjLLQ==
```
Comment (im-klau) [inline .env.example:5]:
```suggestion
APP_KEYS=
```

File: .env.example [hunk 3]
```diff
@@ -1,8 +1,10 @@
...
+API_TOKEN_SALT=A72anEyzTYrvWaoPlRS0ow==
```
Comment (im-klau) [inline .env.example:6]:
```suggestion
API_TOKEN_SALT=
```

File: .env.example [hunk 4]
```diff
@@ -1,8 +1,10 @@
-NODE_ENV=
+NODE_ENV=development
 
-HOST=
-PORT=
+HOST=0.0.0.0
```
Comment (im-klau) [inline .env.example:3]:
```suggestion
HOST=
```

File: .env.example [hunk 5]
```diff
@@ -1,8 +1,10 @@
...
+PORT=1337
 APP_KEYS=
 API_TOKEN_SALT=
-ADMIN_JWT_SECRET=
-JWT_SECRET=
\ No newline at end of file
+ADMIN_JWT_SECRET=Dl756uvppuiAcNOLQCATWQ==
+JWT_SECRET=alvVEqXbgENnnRTd4eYLsw==
+
+MY_HEROKU_URL=https://sterczace-uszy.herokuapp.com
```
Comment (im-klau) [inline .env.example:10]:
```suggestion
```

File: .env.example [hunk 6]
```diff
@@ -1,8 +1,6 @@
-NODE_ENV=
+NODE_ENV=development
```
Comment (im-klau) [inline .env.example:1]:
```suggestion
NODE_ENV=
```

File: .env.example [hunk 7]
```diff
@@ -1,8 +1,6 @@
 NODE_ENV=
 
 HOST=
-PORT=
+PORT=1337
```
Comment (im-klau) [inline .env.example:4]:
```suggestion
PORT=
```

File: .env.example [hunk 8]
```diff
@@ -3,6 +3,4 @@ NODE_ENV=
 HOST=
 PORT=
 APP_KEYS=
-API_TOKEN_SALT=
-ADMIN_JWT_SECRET=
-JWT_SECRET=
\ No newline at end of file
+API_TOKEN_SALT=
```
Comment (im-klau) [inline .env.example:6]:
```suggestion
API_TOKEN_SALT=
ADMIN_JWT_SECRET=
JWT_SECRET=
```

**Review by im-klau — APPROVED** [review]
I forgot this is a public repo and that's why I resolved env issue on my own 😅

**Review by AmandaKr — APPROVED** [review]
🎉

**Pull request merged**